src/lib.rs

```diff
@@ -1,13 +1,17 @@
-#[macro_use] extern crate nom;
+#[macro_use]
```

Comment:
This library more or less models [`descriptor.proto`](https://github.com/google/protobuf/blob/master/src/google/protobuf/descriptor.proto), I'd add a link to that file in the rustdoc.

Reply:
good idea